There is no egg info available for argparse on
Fedora so the requirement can't be satisfied.

diff -Naur python-glanceclient-2012.2.orig/tools/pip-requires python-glanceclient-2012.2/tools/pip-requires
--- python-glanceclient-2012.2.orig/tools/pip-requires	2012-05-24 12:22:17.000000000 +0000
+++ python-glanceclient-2012.2/tools/pip-requires	2012-06-06 12:13:23.258482709 +0000
@@ -1,4 +1,3 @@
-argparse
 httplib2
 prettytable
 
